### PR TITLE
Fix text for overcomplement and add empty alert for bureau AB

### PR DESCRIPTION
**Pull request by scott062**
- overcompliment -> overcomplement
- Empty alert on bureau page is different from CDO empty alert

### CODE REVIEW

File: src/Components/AvailableBidder/EditBidder/EditBidder.jsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const EditBidder = (props) => {
             }}
           >
             <option value="">None listed</option>
-            <option value="OC">OC: Overcompliment</option>
+            <option value="OC">OC: Overcomplement</option>
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/EditBidder/EditBidder.jsx:79]: 🎊

File: src/Components/AvailableBidder/AvailableBidderStats/AvailableBidderStats.jsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ const AvailableBidderStats = (props) => {
                         data.map((entry, index) => <Cell key={`cell-${index}`} fill={COLORS[index % COLORS.length]} />)
                       }
                     </Pie>
-                    <Bar dataKey="OC: Overcompliment" fill="#102f51" />
+                    <Bar dataKey="OC: Overcomplement" fill="#102f51" />
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderStats/AvailableBidderStats.jsx:74]: 🍰

File: src/Components/AvailableBidder/AvailableBidderStats/AvailableBidderStats.jsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ const AvailableBidderStats = (props) => {
     placeholderText,
   } = props;
   const data = [
-    { name: 'OC: Overcompliment', value: 400 },
+    { name: 'OC: Overcomplement', value: 400 },
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderStats/AvailableBidderStats.jsx:17]: 🍬

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 4]
```diff
@@ -108,7 +108,11 @@ const AvailableBidderTable = (props) => {
       <div className="usa-width-two-thirds">
         <Alert
           title="Available Bidders List is Empty"
-          messages={[{ body: 'Please navigate to the CDO Client Profiles to begin searching and adding bidders.' }]}
+          messages={[{
+            body: isCDO ?
+              'Please navigate to the CDO Client Profiles to begin searching and adding bidders.' :
+              'Please wait for CDOs to add available bidders.',
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:114]: `to share available bidders.`?

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍥 🥮 👍

File: src/Components/AvailableBidder/EditBidder/EditBidder.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/AvailableBidder/EditBidder/EditBidder.jsx:79]: 💯

**Pull request merged**